### PR TITLE
added moonraker_channel_updated flag

### PR DESCRIPTION
**Pull request by krautech**
## Description

Flag requirement to ensure moonraker.conf has been updated. This is temporarily in place to ensure the migration over to stable works.

`moonraker_channel_updated: True` if requirements are met

### CODE REVIEW

File: scanner.py [hunk 1]
```diff
@@ -165,7 +165,9 @@ def __init__(self, config):
         }
         self.gcode = self.printer.lookup_object("gcode")
         self.probe_calibrate_z = 0.
-        
+
```
Comment (Jomik) [inline scanner.py:168]: Should we add a `TODO` comment?

**Review by Jomik — CHANGES_REQUESTED** [review]

File: scanner.py (hunk 1 above)
Comment (krautech) [reply scanner.py:168]: more than likely

Comment (krautech) [reply scanner.py:168]: > Should we add a `TODO` comment?

Added

**Review by Jomik — APPROVED** [review]

**Comment by Jomik** [comment]
We don't want this after all.

**Pull request closed without merging**